# Essa é a mensagem que de fato é enviada por uma estratégia para o VSS-Viewer

# Lista dinâmica de vetores de movimentação. Sendo sempre 3, devido a quantidade de robôs de um time
d_pose[3] step_poses

# Lista dinâmica de poses futuras. Sendo sempre 3, devido a quantidade de robôs de um time
d_pose[3] final_poses

# Lista dinâmica de caminhos. Sendo sempre 3, devido a quantidade de robôs de um time
d_path[3] paths
